decosjoin/tests/test_connection.py

```diff
@@ -88,6 +88,11 @@ def test_get_translations(self):
         self.assertIsNone(_get_translation("d", translations))
         self.assertIsNone(_get_translation("Nope", translations))
 
+    def test_to_transition_agreement(self):
+        self.assertIs(to_transition_agreement('Verleend met overgangsrecht'), True)
```

Comment:
Hier kan je beter deze gebruiken:
`assertTrue` en `assertFalse`

of als je zeker wilt weten dat het `False` of `True` is, `assertEqual`

https://docs.python.org/3/reference/expressions.html#is-not